layouts/partials/script.html

```diff
@@ -8,7 +8,7 @@
 <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/2.1.7/js/jquery.fancybox.min.js" integrity="sha256-GEAnjcTqVP+vBp3SSc8bEDQqvWAZMiHyUSIorrWwH50=" crossorigin="anonymous"></script>
 <!--EXTERNAL SCRIPTS END-->
 <!--SCRIPTS-->
-<script src="/js/script-qi9wbxp2ya2j6p7wx1i6tgavftewndznf4v0hy2gvivk1rxgc3lm7njqb6bz.min.js"></script>
+<script src="/js/script-cdeuuojxasyjcchsjy1bjxexo1z16rzagao7rubvrkft2wqj8nyeda192b62.min.js"></script>
```

Comment:
I'm sorry to be nagging but if it is not too much work: Can we exclude this change, as well as the renaming of the javascript files? If nothing was changed, I'd like to keep the currently existing minified script. :)